systemd/systemd.go

```diff
@@ -324,6 +340,7 @@ func (c *Collector) collect(ch chan<- prometheus.Metric) error {
 func (c *Collector) collectUnitState(conn *dbus.Conn, ch chan<- prometheus.Metric, unit dbus.UnitStatus) error {
 	//TODO: wrap GetUnitTypePropertyString(
 	// serviceTypeProperty, err := conn.GetUnitTypeProperty(unit.Name, "Timer", "NextElapseUSecMonotonic")
+	unitStatesName := []string{"active", "activating", "deactivating", "inactive", "failed"}
```

Comment:
Let's keep this as a global variable we don't need to recreate the variable on every function call :)